drivers/wifi/siwx91x/siwx91x_wifi.c

```diff
@@ -303,7 +314,8 @@ static int siwx91x_connect(const struct device *dev, struct wifi_connect_req_par
 	wifi_config.ssid.length = params->ssid_length,
 	memcpy(wifi_config.ssid.value, params->ssid, params->ssid_length);
 
-	ret = sl_wifi_connect(SL_WIFI_CLIENT_INTERFACE, &wifi_config, 0);
+	interface = sl_wifi_get_default_interface();
```

Comment:
Sometime interface is initialized declaration, sometime later in the code. Is it possible to uniformize that?